File: dayThree/javaScript/countDivisorsOfAgivenNumber.js
function countDivisorsOfANumber(number){
    let counter = 0;
    for(let count = 1; count <= number; count++){
        if(number % count == 0){
            counter++; 
        }
    }
    return counter;
}

let number = 20;
countDivisorsOfANumber(number);

File: dayThree/java/CountDivisorsOfAGivenNumber.java
public class CountDivisorsOfAGivenNumber{
    public static void main(String[] args){
        int number = 20;
        System.out.print(countDivisorsOfANumber(number));
    }
    
    public static int countDivisorsOfANumber(int number){
        int counter = 0;
        for(int count = 1; count <= number; count++){
            if(number % count == 0){
               counter++; 
            }
        }
        return counter;
    }
}
